> Lgtm but I'll wait for other review

No doubt unrelated to this PR, but when trying to test it in a codespace:

```
$ gdb
Traceback (most recent call last):
  File "/venv/bin/gdb", line 10, in <module>
    sys.exit(main())
  File "/venv/lib/python3.10/site-packages/gdb_for_pwndbg/gdb.py", line 103, in main
    check_lib_python()
  File "/venv/lib/python3.10/site-packages/gdb_for_pwndbg/gdb.py", line 59, in check_lib_python
    venv_libpath.symlink_to(py_libpath)
  File "/usr/lib/python3.10/pathlib.py", line 1255, in symlink_to
    self._accessor.symlink(target, self, target_is_directory)
PermissionError: [Errno 13] Permission denied: '/usr/lib/x86_64-linux-gnu/libpython3.10.so.1.0' -> '/venv/lib/libpython3.10.so.1.0'
```

--- a/docs/commands/glibc_ptmalloc2_heap/heap.md
+++ b/docs/commands/glibc_ptmalloc2_heap/heap.md
@@ -2,7 +2,7 @@
 # heap
 
 ```text
-usage: heap [-h] [-n COUNT] [-v] [-s] [addr_start] [addr_end]
+usage: heap [-h] [-c COUNT] [-v] [-s] [addr_start] [addr_end]
 
 ```
 
@@ -29,7 +29,7 @@ Range mode can be combined with --count; walking stops when either limit is hit 
 |Short|Long|Help|
 | :--- | :--- | :--- |
 |-h|--help|show this help message and exit|
-|-n|--count|Maximum number of chunks to print.|
+|-c|--count|Maximum number of chunks to print.|
 |-v|--verbose|Print all chunk fields, even unused ones.|
 |-s|--simple|Simply print malloc_chunk struct's contents.|
 

--- a/pwndbg/commands/ptmalloc2.py
+++ b/pwndbg/commands/ptmalloc2.py
@@ -220,13 +220,10 @@ def heap(
         print(message.error("`--count` must be greater than 0."))
         return
 
-    if addr_end is not None:
-        if addr_start is None:
-            print(message.error("`addr_end` requires `addr_start`."))
-            return
-        if addr_end <= addr_start:
-            print(message.error("`addr_end` must be greater than `addr_start`."))
-            return
+    assert addr_end is None or addr_start is not None
+    if addr_end is not None and addr_end <= addr_start:
+        print(message.error("`addr_end` must be greater than `addr_start`."))
+        return
 
     printed_chunks = 0
 

--- a/pwndbg/commands/ptmalloc2.py
+++ b/pwndbg/commands/ptmalloc2.py
@@ -185,7 +185,7 @@ parser.add_argument(
     help="Optional inclusive upper bound for chunk start addresses.",
 )
 parser.add_argument(
-    "-n",
+    "-c",
     "--count",
     type=int,
     default=None,

